MapboxSearch/autofill/src/main/java/com/mapbox/search/autofill/AddressAutofillSuggestion.kt

```diff
@@ -64,8 +61,16 @@ public class AddressAutofillSuggestion internal constructor(
     override fun toString(): String {
         return "AddressAutofillSuggestion(" +
                 "name='$name', " +
-                "formattedAddress='$formattedAddress', " +
-                "coordinate=$coordinate" +
+                "formattedAddress='$formattedAddress'" +
                 ")"
     }
+
+    internal sealed class Underlying : Parcelable {
+
+        @Parcelize
+        data class Suggestion(val suggestion: BaseSearchSuggestion) : Underlying()
+
+        @Parcelize
+        data class Result(val result: BaseSearchResult) : Underlying()
```

Comment:
Result is not needed, address autofill backend will never return result on the first step. You can remove underlying type and use `BaseSearchSuggestion` instead.